lbr_prod_db.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xeu
+docker run --rm --name lbr_rems2 -p 5432:5432 -d postgres:9.6
+docker run --rm --link lbr_rems2 postgres:9.6 /bin/bash -c "while ! psql -h lbr_rems2 -U postgres -c 'select 1;' 2>/dev/null; do sleep 1; done"
+docker run -i --rm --link lbr_rems2 postgres:9.6 psql -h lbr_rems2 -U postgres < resources/sql/lbr_init.sql
```

Comment:
It feels wrong to bundle environment specific SQL files inside the application, in resources/sql directory. That SQL could be inline in this shell script like this:

```
echo "
CREATE USER lbr_rems2_user;
CREATE DATABASE lbr_rems2 OWNER lbr_rems2_user;

CREATE USER lbr_rems2_read;
CREATE USER lbr_rems2_admin;
" | docker run -i --rm --link lbr_rems2 postgres:9.6 psql -h lbr_rems2 -U postgres
```

Reply:
Or should these be in the `rems-deploy` repository?